app/models/comment.rb

```diff
@@ -466,8 +477,15 @@ def count_visible_comments
   end
 
   def check_for_spam?
-    #don't check for spam while running tests or if the comment is 'signed'
-    self.approved = Rails.env.test? || !self.pseud_id.nil? || !Akismetor.spam?(akismet_attributes)
+    self.approved = if %w[staging production].exclude?(Rails.env)
+                      # don't check for spam while in a dev or test environment
+                      true
+                    elsif !self.pseud_id.nil? && !self.pseud.user.should_spam_check_comments?
```

Comment:
Rubocop is trying to make me convert the whole thing to a one-liner, but I'm not convinced that's good for readability